ports/servoshell/Cargo.toml

```diff
@@ -13,6 +13,10 @@ name = "servo"
 path = "main.rs"
 bench = false
 
+[lib]
+name = "servoshell"
+path = "lib.rs"
+
```

Comment:
I don't think it's strictly necessary to move the functions to a lib crate. It probably doesn't make sense to do this unless it is necessary (to write doctests as in the link you gave) or if we are going to be moving a lot of the logic from `servoshell` into a library.